### ISSUE TITLE
'Copy Date' does not respect format

### ISSUE THREAD

**Issue by sanderboom** [mate-desktop/mate-panel#349]
The copied date format is different from the format used in displaying the date in the clock applet.

Also, when changing settings using `dconf` at `org.mate.panel.objects.clock.prefs.format` to `custom` and `org.mate.panel.objects.clock.prefs.custom-format` to e.g. `%a, %d %b %Y %H:%M` this has no effect on the format of Copy Date.

Additional feature request when solving above: separate settings for format for display and 'Copy Date' would be awesome :+1:

**Comment by ericcurtin** [comment]
The only problem here is that the custom-format string is treated as one. And date and time are treated as different strings. Maybe we could change "Copy Time" and "Copy Date" options to be one "Copy Date/Time" option instead and it would use the same custom format as you specified? What do you think? I can implement if you want.

**Comment by sanderboom** [comment]
Awesome.

> Maybe we could change "Copy Time" and "Copy Date" options to be one "Copy Date/Time" option instead and it would use the same custom format as you specified? 

This would be very useful. Maybe add the new option without removing the existing 'Copy Date' and 'Copy Time'? All individual options are useful IMO.

Additionally it would be the right thing to address this:
> The only problem here is that the custom-format string is treated as one. And date and time are treated as different strings. 

Would it be possible to have some new entries in `dconf`?

Similar as `org.mate.panel.objects.clock.prefs.format` but for the copy item:
- `org.mate.panel.objects.clock.prefs.copy-date-format`
- `org.mate.panel.objects.clock.prefs.copy-time-format`

And to specify the actual custom format: 
- `org.mate.panel.objects.clock.prefs.copy-date-format-custom`
- `org.mate.panel.objects.clock.prefs.copy-time-format-custom`

This offers a lot of options for customization, very suitable for a DE like MATE, me thinks.

**Comment by ericcurtin** [comment]
Yeah I can do this. I won't do the Copy Date/Time idea and do what you suggested instead. Only thing is I'm not really a MATE developer, hopefully it gets accepted. I'll create the pull request anyway and we will see!

**Comment by sanderboom** [comment]
Great. I'm sure it will be accepted, thanks!

**Comment by sanderboom** [comment]
@ericcurtin any luck on this one already? :)

**Comment by ericcurtin** [comment]
@sanderboom yeah, I started it but never finished. Unfortunately since this is just a hobby of mine, I don't have any definite finish date. Technically it's not too hard to do, I just must string 4 hours together of my free time. I'll let you know!

**Comment by sanderboom** [comment]
No worries! I can relate to your time problem very well ;)
